keyboards/geonworks/frogmini/fms/config.h

```diff
@@ -0,0 +1,58 @@
+/* 
+Copyright 2022 Gondolindrim 
+ 
+This program is free software: you can redistribute it and/or modify 
+it under the terms of the GNU General Public License as published by 
+the Free Software Foundation, either version 2 of the License, or 
+(at your option) any later version. 
+ 
+This program is distributed in the hope that it will be useful, 
+but WITHOUT ANY WARRANTY; without even the implied warranty of 
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+GNU General Public License for more details. 
+ 
+You should have received a copy of the GNU General Public License 
+along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID 0x2737 // GW for GeonWorks
```

Comment:
Same.

Reply:
Updated for the lower case `0x67777` which seems not to be in use in commit [e2c6eaf](https://github.com/qmk/qmk_firmware/pull/17765/commits/e2c6eafde2028d4d32f7fa1fbe45eb1aabece8f3).